File: papi/lista1/exercicio14.py
a = input("por favor digite o primeiro valor: ")
b = input("por favor digite o segundo valor: ")

# Isto se faz assim em Python
# Em linguagens em que não dá para fazer isto diretamente
# usa-se uma variável temporária
#
# temp = a
# a = b
# b = temp
a, b = b, a

print("Troquei os dois valores ")
print("Agora o primeiro valor é", a)
print("Agora o segundo valor é", b)

File: papi/lista1/exercicio14.kt
package lista1

fun main() {
    println("Iremos trocar 2 valores")
    print("Por favor, digite o primeiro valor: ")
    var valor1 = readLine()!!
    print("Por favor, digite o segundo valor: ")
    var valor2 = readLine()!!
    var tmp = valor2
    valor2 = valor1
    valor1 = tmp
    println("Após a troca, o primeiro valor passou a ser $valor1")
    println("E o segundo valor passou a ser $valor2")
}